### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by chengweige** [naver-ai/class-query-vad#2]
I have finished the training code based on the currently available codes, i test JHMDB dataset with one video，however the loss does not  decrease

<img width="1369" height="322" alt="Image" src="https://github.com/user-attachments/assets/0e2ccba3-a73b-498b-a1f8-82a37c6b7bfd" />

my simplified training code workflow is as follows，I sincerely request your help， thank you.
`   model.train()
    criterion.train()

    #  iterate the train set to train the model
    for step, data in enumerate(data_loader):
        # calculate the time of train process
        data_time.update(time.time() - end)
        # set device to GPU
        device = "cuda:" + str(cfg.DDP_CONFIG.GPU)
        # get image information from dataloader
        samples = data[0]
        
        for t in targets: del t["image_id"]

        # move sample and target to GPU
        samples = samples.to(device)
        targets = [{k: v.to(device) for k, v in t.items()} for t in targets]

        outputs = model(samples)

        # calculate each part loss function
        loss_dict = criterion(outputs, targets)

        # sum each part of loss function and get the loss of the model
        losses = sum(loss_dict[k] * weight_dict[k] for k in loss_dict.keys() if k in weight_dict)

        # set the gradient to zero
        optimizer.zero_grad()
        # back propagation the losses
        losses.backward()

        # use gradient clipping to avoid gradient exploding
        if max_norm > 0:
            torch.nn.utils.clip_grad_norm_(model.parameters(), max_norm)

        # update the gradient
        optimizer.step()

        # update the learning rate
        lr_scheduler.step_update(epoch * len(data_loader) + step)

        # update batch time
        batch_time.update(time.time() - end)
        end = time.time()`
